Makefile.docker

```diff
@@ -0,0 +1,34 @@
+TOPDIR=$(dir $(lastword $(MAKEFILE_LIST)))
+DOCKERFILE_DIR     ?= ./
+DOCKER_REGISTRY    ?= quay.io
```

Comment:
```suggestion
DOCKER_REGISTRY    ?= docker.io
```

Prefer having Docker as default for community users (as in the Strimzi HTTP bridge) even if Strimzi builds use quay.io in the end.